src/cli/build.ts

```diff
@@ -310,7 +310,7 @@ export async function buildModule(reloader?: Reloader, entryPoint = peprTS, embe
 
     return { ctx, path, cfg, uuid };
   } catch (e) {
-    console.error(e.message);
+    console.error(`Error building module: ${e.message}`);
```

Comment:
```suggestion
    console.error(e, `Error building module`);
```

Reply:
All messages updated to display complete `e`